grakn-test/test-integration/src/test/java/ai/grakn/test/benchmark/AddBenchmark.java

```diff
@@ -39,7 +49,15 @@ public void tearDown() {
     }
 
     @Benchmark
-    public void inserts() {
+    public void addEntity() {
         entityType.addEntity();
     }
+
+    @Benchmark
+    public void addRelation() {
+            Entity entity1 = entityType.addEntity();
+            Entity entity2 = entityType.addEntity();
+            relationshipType.addRelationship().addRolePlayer(role1, entity1).addRolePlayer(role2, entity2);
+            graph.commit();
```

Comment:
Since we have `AddWithCommitBenchmark ` which already contains the above I am guessing having this commit line here is a booboo